### DISCUSSION TITLE
Aggiornamento 2021.7.3 non va a buon fine

### DISCUSSION THREAD

**Discussion by calz64** [indomus/forum#526] [HUB: Home Assistant]
E' la seconda volta che devo reinstallare HA, l'aggiornamento HA-Core 2021.7.3 si pianta. 
Qualcuno ha riscontrato lo stesso problema?

**Comment by snakuzzo** [comment]
Innanzi tutto dovresti specificare il tipo di installazione che hai.
Detto ciò, ci sono i log da guardare. I problemi non si risolvono reinstallando di continuo.

**Reply by calz64** [reply]
Innanzitutto grazie della risposta, l'istallazione è hassOS, e il log non riesco a guardarli perchè una volta attivato l'aggiornamento che si vede nell'immagine allegata, HA non è più accessibile, il sistema non si avvia più.
![Cattura](https://user-images.githubusercontent.com/84615011/126024953-19996035-da67-4340-9a6e-8193965f9d70.JPG)

**Reply by NdR91** [reply]
Il sistema non si avvia più, intendi da GUI o anche in ssh non accedi?

**Reply by calz64** [reply]
si da gui,  non ho pensato alla connessione ssh, però devo dire che non sarei molto in grado di muovermi da riga di comando. alla fine ho reinstallato, nel ricaricare lo snapshot devo dire che motioneye è andato perso, mah? il resto è a posto.
Sono quindi l'unico che non fa l'aggiornamento, ho caricato il sistema su ssd esterno con boot da ssd potrebbe essere quello? ho letto che il boot da ssd non è ancora consigliabile, ho letto la guida per fare il boot da microsd mettendo OS e dati su ssd esterno e non è proprio una passeggiata! Qualcuno sa consigliarmi?

**Reply by 50494554524F** [reply]
è semplicissimo...

https://github.com/home-assistant/operating-system/blob/39babd53685d352c3f4a6d7d8fabe43e27ccc88d/Documentation/partition.md#data

**Reply by calz64** [reply]
In realtà sempliccissimo non è, ti propongo queste guide che ti seguono passo passo, credo che sia concettualmente la stessa cosa (usa le stesse funzioni) però ti guida meglio e le insidie ci sono.
**LINK RIMOSSO
LINK RIMOSSO**

**Reply by snakuzzo** [reply]
Link ad altri siti non sono ammessi. Rimuovo.
LEGGETE IL REGOLAMENTO 

P.S. Pensiero personale...se hai seguito guide di altri siti, chiedi info su quei quei siti...no ?

**Comment by calz64** [comment]
Ho provato ancora a fare l'aggiornamento, saltando alla 2021.8.6, ma il risultato apparentemente non cambia, però ho provato l'accesso con ssh e in effetti cosa scopro:  
- che accedo ad HA e che l'aggiornamento è stato fatto!!! 
-  che in effetti il sistema funziona, ad es. se do comandi vocali con alexa le cose funzionano (EDIT: ho detto una stupidata i comandi con alexa funzionano anche con HA spento!!!!!!  **Quindi non ho capito nulla..**. pensavo che alexa richiamasse automazioni di HA es. chiudi tapparelle con shelly 2.5, ma in realtà può funzionare anche senza HA)
- quindi quello che succede è che non riesco ad accedere : (vedi EDIT sotto)_con interfaccia grafica e credo che sia un problema effettivamente di aver spostato su SSD il tutto_. 
EDIT: scusate ultima scoperta (lascio i passaggi per confessare il mio livello basso di conoscenza), scopro che posso accedere all'interfaccia grafica in localhost (funziona tutto) quindi a questo punto il solo problema è:  - duck DNS?  perché è saltato l'accesso da remoto? nei log non riesco a vedere l'errore e nei registri di Duck e NGINX tutto sembra funzionare nessun errore????

**Reply by snakuzzo** [reply]
Hai letto la parte relativa a "reverse proxy" ?
https://indomus.it/news/aggiornamento-2021-7-per-home-assistant-core/

**Reply by calz64** [reply]
Chiedo scusa, finalmente ho letto regolamento, non è così impegnativo, 2 punti e due punti che ho bellamente violato - sorry. 
Però il primo punto non lo capisco appieno, è davvero un forte limite al mutuo aiuto non poter segnalare una guida o una soluzione che può svoltare. Grazie  prometto di non violare più.

**Reply by snakuzzo** [reply]
Il regolamento è quello...link a siti terzi non sono ammessi (così come avviene in tante altre community).
Ad ogni modo non siamo qui a discutere il regolamento. Quando entri in una community o altrove...entri...accetti il regolamento e lo rispetti. Se non lo accetti esci e vai altrove. Mi sembra un discorso abbastanza coerente.

**Reply by calz64** [reply]
si va bene , spero di non averti offeso, era solo una riflessione, rimango e sto al mio posto don't worry. Grazie comunque del link prezioso mi sembra di aver capito l'errore che ho commesso.

**Reply by snakuzzo** [reply]
Nessuna offesa :D
Era per chiarire.
Tornando al tema principale...hai visto l'articolo ? Se usi un reverse proxy devi aggiungere un blocchettino in configurazione

**Reply by calz64** [reply]
perfetto, tutto è tornato a posto, ho aggiunto un tassellino alla mia ignoranza. Rimane possibile per ora avere HA su SSD senza problemi

**Reply by snakuzzo** [reply]
Curiosità...che era ?
Quanto all'SSD...certo che puoi passarlo su SSD

**Reply by calz64** [reply]
Il tema è questo:
 Ho già passato HA su SSD e tutto per ora funziona (a parte questo post), c'è però chi suggerisce di lasciare S.O. e Core su microsd e spostare su SSD solo mnt/data con procedura consigliata anche da Pietro post precedente (dai link che avevo postato la procedura a mio parere risulta complessa, per nulla una passeggiata) 
```
è semplicissimo...
https://github.com/home-assistant/operating-system/blob/39babd53685d352c3f4a6d7d8fabe43e27ccc88d/Documentation/partition.md#data
```
Ma il suggerimento di lasciare boot su microsd è una discussione anche degli sviluppatori, a quanto ho capito per raspi4, perchè il boot diventa complesso/fragile, ed io pensavo proprio che l'aggiornamento fatto (il motivo di questo tread) mi avesse corrotto il boot
https://www.home-assistant.io/faq#is-usb-boot-for-the-raspberry-pi-4-supported

Comunque io per ora continuo ad utilizzare la mia installazione HA su SSD, compreso il boot, e spero che non dia problemi, o che la problematica citata venga sistemata dai prossimi aggiornamenti.
